internal/authtoken/repository.go

```diff
@@ -361,6 +361,55 @@ func (r *Repository) DeleteAuthToken(ctx context.Context, id string, opt ...Opti
 // "issued".  If the token has already been issued, an error is returned with a
 // nil token.  If no token is found for the tokenRequestId an error is returned
 // with a nil token.
+//
+// Note: no oplog entries are created for auth token operations (this is intentional).
 func (r *Repository) IssueAuthToken(ctx context.Context, tokenRequestId string) (*AuthToken, error) {
-	panic("to-do")
+	const op = "authtoken.(Repository).IssueAuthToken"
```

Comment:
Now that issuing the auth token isn't done at the same time as creation of the auth token, do we need to update the auth token expiration time or last accessed time based on the issue time?

Reply:
I had asked about expiry earlier, and we sort of aligned on need need to update it when it's issued.   Last time accessed should be updated I guess.  I'm thinking, there should be some trigger that bumps the timestamp when the token is updated to issued. Make sense?